docs/vcd-recoding.md

```diff
@@ -1,5 +1,9 @@
 # VCD Recoding
 
+:::{attention}
+Deprecated and will be removed in GTKWave 4.
+:::
+
```

Comment:
This is still used by GTKWave 4. The VCD loader that didn't use recoding has been removed.
```suggestion
```